### PR TITLE
Feat/meetup test

### PR DESCRIPTION
**Pull request by AniaMakes**
## Description - [Trello ticket 611](https://trello.com/c/At97k5x8/611-meetup-lambda-test)

Took the tests this function had a year ago (https://github.com/yldio/website-contentful-data-transfer) and worked them into the function's current state. 

## Checklist

- [ ] Updating or creating a new page? Consider any SEO requirements such as:
  - [ ] Does the page have an `h1` tag?
  - [ ] Does the page have a correctly formed meta title?
  - [ ] Does the page have a correctly formed meta description?
  - [ ] Does the page have a unique `og:image` tag (if required)?
- [ ] checked that this PR resolves the spec provided from trello / this description

If you're unsure how to check these SEO requirements please reach out to your colleagues for help!

### CODE REVIEW

File: src/functions/helperFiles/meetupHelpers.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    arrayOfMeetups.forEach(meetup => {
+        const thisMeetup = {
+            meetupId: meetup.id,
+            name: meetup.name,
+            url: meetup.link,
+            urlname: meetup.urlname,
+            nextEvent: meetup.next_event ? meetup.next_event.id : 0
+        }
+
+        outputArray.push(thisMeetup)
```
Comment (GBouffard) [inline src/functions/helperFiles/meetupHelpers.js]: No need to define `const thisMeetup` if used once.
```
outputArray.push({
  meetupId: meetup.id,
  name: meetup.name,
  url: meetup.link,
  urlname: meetup.urlname,
  nextEvent: meetup.next_event ? meetup.next_event.id : 0
})
```

Comment (GBouffard) [reply src/functions/helperFiles/meetupHelpers.js]: Actually better...for each with a push...we can simply `.map`
```
const processMeetupData = arrayOfMeetups => arrayOfMeetups.map(meetup => ({
  meetupId: meetup.id,
  name: meetup.name,
  url: meetup.link,
  urlname: meetup.urlname,
  nextEvent: meetup.next_event ? meetup.next_event.id : 0
}))
```

File: src/functions/helperFiles/meetupHelpers.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                eventObject.description.includes('EVENT DETAILS')
+                ? striptags(
+                    eventObject.description
+                        .split('EVENT SUMMARY:')[1]
+                        .split('EVENT DETAILS')[0]
+                ).trim()
+                : 'For more information, please visit the Meetup page'
+    }
+
+    return outputObject
```
Comment (GBouffard) [inline src/functions/helperFiles/meetupHelpers.js]: No need to define a `outputObject` if it's returned right away. we can just return the object.

File: src/functions/helperFiles/meetupHelpers.js (hunk 1 above)
Comment (AniaMakes) [reply src/functions/helperFiles/meetupHelpers.js]: .map is magic!

**Pull request closed without merging**